Marcio/1dot content strategy pcx2509 (#2424)
cloudflare/cloudflare-docs: products/1.1.1.1/src/content/faq.md

# FAQ

> Below you will find answers to our most commonly asked questions on 1.1.1.1 DNS resolver. If you cannot find the answer you are looking for, heading over to our [community page](https://community.cloudflare.com/) is always a good idea.

Reviewer: ```suggestion
Below you will find answers to our most commonly asked questions on 1.1.1.1 DNS resolver. If you cannot find the answer you are looking for, visit our [community page](https://community.cloudflare.com/).
```
